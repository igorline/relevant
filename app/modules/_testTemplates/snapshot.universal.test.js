import React from 'react';
import renderer from 'react-test-renderer';
import AvatarBox from 'modules/user/avatarbox.component';
import { MemoryRouter } from 'react-router-dom';

const user = { _id: 123, handle: 'handle', name: 'Name', relevance: { pagerank: 10 } };
const postTime = new Date().toISOString();

jest.mock('react-redux', () => ({ useDispatch: () => () => {}, connect: el => el }));

const props = {
  user,
  relevance: false,
  postTime,
  setSelected: jest.fn()
};

test('Snapshot AvatarBox Styled', () => {
  const tree = renderer
    .create(
      <MemoryRouter>
        <AvatarBox {...props} />
      </MemoryRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
