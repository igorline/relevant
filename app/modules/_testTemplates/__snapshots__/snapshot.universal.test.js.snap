// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot AvatarBox Styled 1`] = `
<div
  className="css-view-1dbjc4n"
>
  <a
    className="sc-AxjAm hBufrA ulink-mock"
    href="/user/profile/handle"
    hu={0}
    inline={0}
    onClick={[Function]}
    styles=""
  >
    <div
      className="css-view-1dbjc4n"
      style={
        Object {
          "WebkitBoxDirection": "normal",
          "WebkitBoxOrient": "horizontal",
          "WebkitFlexDirection": "row",
          "display": "flex",
          "flexDirection": "row",
          "msFlexDirection": "row",
          "position": "relative",
        }
      }
    >
      <div
        className="css-view-1dbjc4n"
        style={
          Object {
            "display": "flex",
            "position": "relative",
          }
        }
      >
        <div
          className="css-view-1dbjc4n"
          style={
            Object {
              "WebkitFlexBasis": "auto",
              "borderBottomLeftRadius": "1rem",
              "borderBottomRightRadius": "1rem",
              "borderTopLeftRadius": "1rem",
              "borderTopRightRadius": "1rem",
              "display": "flex",
              "flexBasis": "auto",
              "height": "2rem",
              "msFlexPreferredSize": "auto",
              "overflowX": "hidden",
              "overflowY": "hidden",
              "position": "relative",
              "width": "2rem",
              "zIndex": 0,
            }
          }
        >
          <div
            className="css-view-1dbjc4n"
            style={
              Object {
                "backgroundColor": "rgba(0,0,0,0.00)",
                "backgroundImage": "url(\\"default_user.jpg\\")",
                "backgroundPosition": "center",
                "backgroundRepeat": "no-repeat",
                "backgroundSize": "cover",
                "bottom": "0px",
                "height": "100%",
                "left": "0px",
                "position": "absolute",
                "right": "0px",
                "top": "0px",
                "width": "100%",
                "zIndex": -1,
              }
            }
          />
          <img
            alt=""
            className="css-accessibilityImage-9pa8cd"
            draggable={false}
            src="default_user.jpg"
          />
        </div>
      </div>
      <div
        className="css-view-1dbjc4n"
        style={
          Object {
            "WebkitAlignItems": "flex-start",
            "WebkitBoxAlign": "start",
            "alignItems": "flex-start",
            "display": "flex",
            "marginLeft": "0.5rem",
            "msFlexAlign": "start",
            "position": "relative",
          }
        }
      >
        <div
          className="css-text-901oao"
          dir="auto"
          style={
            Object {
              "position": "relative",
            }
          }
        >
          <span
            className="css-text-901oao css-textHasAncestor-16my406 css-textOneLine-bfa6kz"
            dir="auto"
            style={
              Object {
                "WebkitTextDecorationColor": "rgba(0,0,0,1.00)",
                "color": "rgba(0,0,0,1.00)",
                "fontFamily": "HelveticaNeue",
                "fontSize": "0.875rem",
                "lineHeight": "1.2249999999999999rem",
                "maxWidth": "6rem",
                "overflowX": "hidden",
                "overflowY": "hidden",
                "textDecorationColor": "rgba(0,0,0,1.00)",
                "whiteSpace": "nowrap",
              }
            }
          >
            Name
          </span>
        </div>
        <div
          className="css-text-901oao"
          dir="auto"
          style={
            Object {
              "color": "rgba(128,128,128,1.00)",
              "fontFamily": "HelveticaNeue",
              "fontSize": "0.75rem",
              "lineHeight": "1.0499999999999998rem",
              "marginTop": "0.125rem",
            }
          }
        >
          @handle
           
           â€¢ A Few Seconds Ago
        </div>
      </div>
    </div>
  </a>
</div>
`;
